src/App.js

```diff
@@ -5,135 +5,166 @@
+import CreateTokenAmount from './screens/CreateTokenAmount';
+import CreateTokenConfirm from './screens/CreateTokenConfirm';
+import CreateTokenDetail from './screens/CreateTokenDetail';
 import RegisterTokenManual from './screens/RegisterTokenManual';
-import Settings from './screens/Settings';
-import TokenDetail from './screens/TokenDetail';
-import UnregisterToken from './screens/UnregisterToken';
-import PinScreen from './screens/PinScreen';
+import CreateTokenName from './screens/CreateTokenName';
+import CreateTokenSymbol from './screens/CreateTokenSymbol';
 import About from './screens/About';
 import Security from './screens/Security';
 import PushNotification from './screens/PushNotification';
 import ChangePin from './screens/ChangePin';
-import ResetWallet from './screens/ResetWallet';
-import Dashboard from './screens/Dashboard';
-import LoadHistoryScreen from './screens/LoadHistoryScreen';
-import CreateTokenDepositNotice from './screens/CreateTokenDepositNotice';
-import CreateTokenName from './screens/CreateTokenName';
-import CreateTokenSymbol from './screens/CreateTokenSymbol';
-import CreateTokenAmount from './screens/CreateTokenAmount';
-import CreateTokenConfirm from './screens/CreateTokenConfirm';
-import CreateTokenDetail from './screens/CreateTokenDetail';
-import LoadWalletErrorScreen from './screens/LoadWalletErrorScreen';
-import { WALLET_STATUS } from './sagas/wallet';
-
+import PaymentRequestDetail from './screens/PaymentRequestDetail';
+import ChangeToken from './screens/ChangeToken';
+import TokenDetail from './screens/TokenDetail';
+import UnregisterToken from './screens/UnregisterToken';
+import ReceiveScreen from './screens/Receive';
+import Settings from './screens/Settings';
 
-const InitStack = createStackNavigator(
-  {
-    WelcomeScreen,
-    InitialScreen,
-    NewWordsScreen,
-    LoadWordsScreen,
-    BackupWords,
-    ChoosePinScreen,
-  },
-  {
-    initialRouteName: 'WelcomeScreen',
-    headerMode: 'none',
-  }
-);
+/**
+ * This Stack Navigator is exhibited when there is no wallet initialized on the local storage.
+ */
+const InitStack = () => {
+  const Stack = createStackNavigator();
+  return (
+    <Stack.Navigator
+      initialRouteName='Welcome'
+      screenOptions={{
+        headerShown: false,
+      }}
+    >
+      <Stack.Screen name='WelcomeScreen' component={WelcomeScreen} />
+      <Stack.Screen name='InitialScreen' component={InitialScreen} />
+      <Stack.Screen name='NewWordsScreen' component={NewWordsScreen} />
+      <Stack.Screen name='LoadWordsScreen' component={LoadWordsScreen} />
+      <Stack.Screen name='BackupWords' component={BackupWords} />
+      <Stack.Screen name='ChoosePinScreen' component={ChoosePinScreen} />
+    </Stack.Navigator>
+  );
+};
 
-const DashboardStack = createStackNavigator(
-  {
-    Dashboard,
-    MainScreen,
-  },
-  {
-    initialRouteName: 'Dashboard',
-    headerMode: 'none',
-  },
-);
+const DashboardStack = () => {
```

Comment:
🚲 I think we should have a document string here like we see in InitStack.

Reply:
Done on [ae88d09](https://github.com/HathorNetwork/hathor-wallet-mobile/pull/286/commits/ae88d09c6cdb386a0025367f7f7a6713205f4e72)